test/system/005-info.bats

```diff
@@ -258,4 +258,18 @@ EOF
     run_podman $safe_opts system reset --force
 }
 
+@test "podman - warning message on launching Podman with non-tmpfs tmpdir" {
+    skip_if_remote "Check requires a local Libpod"
+
+    # We need a directory that is not tmpfs to put Podman in
+    OLDTMPFS=$PODMAN_SUPPRESS_TMPFS_WARNING
+    unset PODMAN_SUPPRESS_TMPFS_WARNING
+
+    # Anything that creates a Libpod should trigger the warning.
+    run_podman info
+    assert "$output" =~ "The following temporary files directories are not on a tmpfs filesystem"
+
```

Comment:
Might I also suggest
```bash
    tmpfs_tmp=$(mktemp -d /dev/shm)
    run_podman --tmpdir=$tmpfs_tmp/tmp --runroot=$tmpfs_tmp/runroot --root=$tmpfs_tmp/root info
    assert "$output" !~ "are not on a tmpfs"
```

Reply:
Any reason to not just `mktemp` and then mount a tmpfs over that directory?

Reply:
Laziness? And, will that work rootless? I've never seen a system without `/dev/shm` (at least not in the last 20 years)

Reply:
Fair point on rootless